src/components/dashboard/sidebar/SubsidiesSummary.jsx

```diff
@@ -40,16 +61,17 @@ const SubsidiesSummary = ({ className, showSearchCoursesCta }) => {
             />
           )
         }
-        {offersCount > 0 && (
+        {(offersCount > 0 || pendingCouponCodeRequests.length > 0) && (
           <OfferSummaryCard
             offersCount={offersCount}
+            couponCodeRequestsCount={pendingCouponCodeRequests.length}
             className="mb-3"
           />
         )}
         {!disableSearch && showSearchCoursesCta && (
           <Link
```

Comment:
nit: I believe this could be written as the following to avoid needing to worry about the `.btn` or `.btn-*` classes.

```jsx
import { Button } from '@edx/paragon';

<Link
  to={`/${slug}/search`}
  as={Button}
  variant={ctaButtonVariant}
  block
>
  {CATALOG_ACCESS_CARD_BUTTON_TEXT}
</Link>
```